app/data/deep-dive/_features/contributors-table/_components/filter-data/filter-data.tsx

```diff
@@ -44,6 +46,12 @@ export function FilterData() {
           selectedCategories={filters.categoryIds}
           onSelect={categories => setFilters({ categoryIds: categories })}
         />
+        <ActivityStatusesFilter
```

Comment:
you can type the select to return on ContributorActivityStatusesUnion instead of casting